src/odemis/gui/xmlh/resources/panel_fastem_user_settings.xrc

```diff
@@ -98,7 +98,7 @@
                 <tooltip>Select scintillators containing sample.</tooltip>
```

Comment:
@pieleric, the `FastEMSelectionPanel` specifically for the user settings panel takes time to show. If I click/unclick the chamber button it show instantly. Do you know what could be the reason?

Reply:
No specific idea. Mabe a bug in `FastEMSelectionPanel`. Or maybe it's not initialized with the scintillator data until LOADING the chamber?